### PR TITLE
feat(wedof):  11 actions / 8 triggers

### PR DESCRIPTION
**Pull request by vbarrier**
Hello, 

We would like to contribute to a new Piece from our SaaS platform Wedof. We are commited to automate administrative work around the training / certification domain. 

We would be happy to use Activepieces as one of our solutions to help our customers to automate their work.

### CODE REVIEW

**Review by abuaboud — APPROVED** [review]

File: packages/pieces/community/wedof/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    }
+  },
+});
+
+export const wedof = createPiece({
+    displayName: 'Wedof',
+    auth: wedofAuth,
+    description: 'Automatisez la gestion de vos dossiers de formations (CPF, EDOF, Kairos, AIF, OPCO et autres)',
+    minimumSupportedRelease: '0.20.0',
+    logoUrl: 'https://www.wedof.fr/assets/images/logo/logo.svg',
```
Comment (abuaboud) [inline packages/pieces/community/wedof/src/index.ts]: Can you please change this to `https://cdn.activepieces.com/pieces/wedof.svg`

**Review by abuaboud — COMMENTED** [review]
Thank you so much for great PR!

Can you run lint command and we are good to go

**Comment by abuaboud** [comment]
@allcontributors please add @vbarrier for plugin

File: packages/pieces/community/wedof/src/index.ts (hunk 1 above)
Comment (vbarrier) [reply packages/pieces/community/wedof/src/index.ts]: I have updated the pull request

Thank you!

**Comment by vbarrier** [comment]
I made a lint command on the piece code

**Review by WILAMben — APPROVED** [review]

**Comment by vbarrier** [comment]
Any update @abuaboud?

**Comment by vbarrier** [comment]
Thank you!

**Pull request merged**